source/includes/usage-examples/write-code-examples.java

```diff
@@ -0,0 +1,77 @@
+# start-insert-one
+Document document = new Document("<field name>", "<value>");
+Mono<InsertOneResult> result = Mono.from(collection.insertOne(document));
+
+System.out.printf("Inserted 1 document with ID %s%n.", 
+      result.block().getInsertedId());
+# end-insert-one
+
+# start-insert-multiple
+Document doc1 = new Document("<field name>", "<value>");
+Document doc2 = new Document("<field name>", "<value>");
+
+List<Document> documents = Arrays.asList(doc1, doc2);
+
+Mono<InsertManyResult> result = Mono.from(collection.insertMany(documents));
+
+System.out.printf("Inserted documents with IDs %s%n.",
+      result.block().getInsertedIds());
+# end-insert-multiple
+
+# start-update-one
+Mono<UpdateResult> result = Mono.from(collection.updateOne(
+      eq("<field name>", "<value>"),
+      set("<field name>", "<new value>")));
+
+System.out.printf("Inserted %s document with ID %s%n.",
+      result.block().getModifiedCount(), result.block().getUpsertedId());
```

Comment:
One is not supposed to subscribe to our publishers more than once (if you do that, it may either fail with exceptions that depend on the situation, or happen to succeed, which is even worse). If you need to call more than one method on the `BulkWriteResult` returned by `result.block` here, then store a reference to that `BulkWriteResult` in a variable and call methods via that variable.

This is applicable to more than one place in `write-code-examples.java`.

Reply:
Calling `BulkWriteResult blockedResult` is really weird: the result was never blocked. I propose calling naming this variable similarly to its type `bulkWriteResult`, as it seems fine in this case.

Reply:
The variable was renamed to `result`, but then it is used as if it were named `bulkResult`. As a result, the code does not compile.